AGPs/AGP-1H4.md

```diff
@@ -0,0 +1,46 @@
+---
+AGP: tbd
+Title: Improved Deployment Processes and Testnet Support
+Author: @lkngtn (@1hive)
+Status: Stage II
+Track: Finance
+Created: 2019-11-12
+---
+
+# **AGP-TBD: Improved Deployment Processes and Testnet Support**
+
+## **Address of the transfer recipient**
+
+ `0x9213737fbb72d8f924fca55d508ae2c004413d64` Vault of the [dandelion project organization](https://mainnet.aragon.org/#/1hive)
+
+## **Amount of the transfer**
+
+15,000 DAI Due upon approval
+5,000 ANT Due upon successful delivery
+
+## **Number and frequency of transfers if recurring**
+
+2 (One upon approval, one upon successful delivery).
+For the purpose of budgeting, the full amount should be debited against this quarter's budget.
+
+## **Purpose of the transfer**
+
+Testnet deployment is an important part of the development process. However, it is currently difficult to setup an Aragon Development environment outside of the Ganache snapshot provided by the Aragon CLI. 1Hive has experienced this issue working on Apiary, where we need to have Aragon running in a local test environment with a real node. As well as when working on [Flora](https://devpost.com/software/flora-sidechain-discount-transactions-q8usv3)
+
+Additionally, there have been instances where the lack of support for Goerli, Ropsten, and Kovan testnets has produced friction between projects that would like to integrate Aragon into their protocols or dapps.
+
+To improve this we would like to improve deployment infrastructure and documentation in order to make it easy to deploy and maintain the Aragon environment on multiple blockchain networks.
+
+In exchange for the funding requested we will:
+* Make it easier to deploy the Aragon environment to a local chain in order to support development workflows that need access to a full node as opposed to a ganache snapshot.
```

Comment:
Do we add this to aragonCLI or as a standalone tool?

Reply:
> Do we add this to aragonCLI or as a standalone tool?

I think a standalone script is probably fine, not opposed to using or adding to the CLI but wouldn't want to specify that given that the CLI seems to be changing a lot right now.